poll.go

```diff
@@ -440,7 +440,7 @@ func (h *Headscale) scheduledPollWorker(
 		case <-updateCheckerTicker.C:
 			// Send an update request regardless of outdated or not, if data is sent
 			// to the node is determined in the updateChan consumer block
-			n, _ := m.toNode()
+			n, _ := m.toNode(true)
```

Comment:
I really dont like that arguments cant have descriptors.

Reply:
Me neither. Any idea here? I reckon we should keep Tailscale.com's behaviour of not exposing the routes across shared machines...

Reply:
No, sadly, I think other projects use a list object and functions with name, but it seems too much of a hassle for this.